docs/data/catalog/platform/gc-forms/templates.md

```diff
@@ -49,56 +49,56 @@ Here's a descriptive list of the fields in each table:
 
 ### Table: platform_gc_forms_template
```

Comment:
what about you ask gpt to format this as a table? looks neat https://github.com/cds-snc/data-lake/blob/main/docs/data/catalog/bes/crm/salesforce.md